### PR TITLE
create function code

### PR DESCRIPTION
**Pull request by MattyOram**

### CODE REVIEW

File: src/__pycache__/range.cpython-310.pyc
Comment (RFurze) [inline src/__pycache__/range.cpython-310.pyc:1]: Delete this file Matty. Create a .gitignore and include __pychace__

File: src/__pycache__/time_range.cpython-310.pyc
Comment (RFurze) [inline src/__pycache__/time_range.cpython-310.pyc:1]: Delete this file Matty. Create a .gitignore and include __pychace__

**Review by RFurze — CHANGES_REQUESTED** [review]

File: src/__pycache__/range.cpython-310.pyc
Comment (MattyOram) [reply src/__pycache__/range.cpython-310.pyc:1]: done boss

File: src/__pycache__/time_range.cpython-310.pyc
Comment (MattyOram) [reply src/__pycache__/time_range.cpython-310.pyc:1]: done

**Review by RFurze — APPROVED** [review]

**Pull request merged**